.github/workflows/auto-update.yml

```diff
@@ -82,6 +111,12 @@ jobs:
         run: |
           EXTERNAL_IMAGE=$(yq '.images[] | select(.source | contains("kube-rbac-proxy")) | .source' external-images.yaml)
           scripts/await_image.sh "${EXTERNAL_IMAGES_REPO}/${EXTERNAL_IMAGE}"
+
+      - name: Checkout code after chart update and external images sync
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - run: git pull
```

Comment:
I'm not sure if this will work without giving explicitly the latest commit ref after merging the PR with image-syncer.

Reply:
afaik, the checkout action triggered from the workflow on the schedule on the main branch should fetch the origin of the repo and checkout to the main branch. Then we have git pull to get the merged changes. I will quick fix the workflow in another PR in case the flow goes differently.